portfolio/src/main/java/com/google/sps/servlets/JsonServlet.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```

Comment:
nit: update comment since this is /greeting

Reply:
Resolved. Change the whole JsonServlet.java into greetingServlet.java file instead